test/controllers/HomeControllerSpec.scala

```diff
@@ -21,22 +21,18 @@
 
 package controllers
 
-import org.scalatestplus.play._
-import org.scalatestplus.play.guice._
 import play.api.mvc.{ AnyContentAsEmpty, DefaultActionBuilder, DefaultMessagesActionBuilderImpl, DefaultMessagesControllerComponents }
 import play.api.test._
 import play.api.test.Helpers._
 import uk.gov.nationalarchives.omega.editorial.controllers.{ HomeController, SessionKeys }
-import uk.gov.nationalarchives.omega.editorial.models.session.Session
+import support.BaseSpec
 
 /** Add your spec here. You can mock out a whole application including requests, plugins etc.
   *
   * For more information, see https://www.playframework.com/documentation/latest/ScalaTestingWithScalaTest
   */
-class HomeControllerSpec extends PlaySpec with GuiceOneAppPerTest with Injecting {
+class HomeControllerSpec extends BaseSpec {
```

Comment:
I wanted to not start the JMS echo server in unit tests, so I changed these unit specs to use a version of the application without the StartupModule.